robotidy/rich_console.py

```diff
@@ -0,0 +1,51 @@
+try:
+    from rich.console import Console
+    from rich.markdown import Markdown
+    from rich.table import Table
+    RICH_SUPPORTED = True
+except ImportError:
+    RICH_SUPPORTED = False
+    import re
+    import click
+
+
+if RICH_SUPPORTED:
+    console = Console()
+else:
+    class Console:
+        @staticmethod
+        def print(msg):
+            msg = re.sub(r"\[[^\]]+\]", "", str(msg))
+            click.echo(msg)
+
+    class Markdown:
+        def __init__(self, documentation, **kwargs):
+            self.documentation = documentation
+
+        def __str__(self):
+            return self.documentation
+
+    class Table:
```

Comment:
If we don't have rich installed we're using this class to create table-like outputs (for example for list of transformers). It only provides the basic output - but doesn't need any extra dependency.